### PR TITLE
Structure To Do Taks List Project

### PR DESCRIPTION
**Pull request by ALPANNA-GH**
- setup webpack
- Well file structure
- Structure `HTML`, `CSS`, `JS` files as required
- Build tasks class
- Dynamically load tasks
- Styling Elements

### CODE REVIEW

File: src/scripts/Tasks.js [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
-
 export class Tasks {
-  constructor() {
+  constructor () {
     this.taskList = [
       { index: 1, desc: 'task1', done: false },
       { index: 2, desc: 'task2', done: false },
-      { index: 3, desc: 'task3', done: false }
+      { index: 3, desc: 'task3', done: false },
+      { index: 4, desc: 'task4', done: false },
```
Comment (Olamarx) [inline src/scripts/Tasks.js:7]: - [ ] I will encourage you to always prioritize project requirements to your personal thoughts, the project requirement states that you use description, not desc, also completed not done. Kindly fix those up, outside these, you did a wonderful work.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @ALPANNA-GH, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Array data ✔️ 
- Array iteration ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/scripts/Tasks.js (hunk 1 above)
Comment (ALPANNA-GH) [reply src/scripts/Tasks.js]: ok all done

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ALPANNA-GH , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**